### PR TITLE
FFP and Cost Allocation export view issue

### LINKED ISSUE #2385: FFP and budget page data from Activity 1 is being shown in every Activity in the export view
The FFP and budget page data from Activity 1 is being shown in every Activity in the export view

This is easy to reproduce by going to any APD that has more than one Activity and viewing the export page. You'll see in each activity the Cost Allocation and Budget for FFY X starts with the Key Personnel role in the first row which should only be in the admin Activity, and then all of the rest of the data is from Activity 1 as well. 

This is happening for all fields/tables on the FFP and Budget page. 

### This task is done when...
 - [ ] The correct Activity's FFP and Budget page data is showing in the export view

### PR DESCRIPTION
**Pull request by thetif**
fixed issue with activities having the wrong FFP and Cost Allocation tables on the export view; added react testing library
closes #2385 

### This pull request changes...

- on the export view, the budget tables under each activity should now show the right data and not just the first table data

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
~- [ ] The change has been documented~
  ~- [ ] Associated OpenAPI documentation has been updated~
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/containers/viewOnly/activities/Activity.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ const Activity = ({ activity, activityIndex }) => {
       </h3>
       <CostAllocateFFP
         aKey={activity.key}
-        activityIndex={0}
```
Comment (radavis) [inline web/src/containers/viewOnly/activities/Activity.js:245]: Well, that'll do it, won't it.

File: web/src/shared/apd-testing-library.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      ),
+    store,
+    history
+  };
+};
+
+// Re-export all of the react-testing-library functions,
+// so that you don't have to import both libraries
+export * from '@testing-library/react'; // eslint-disable-line import/no-extraneous-dependencies
+export { renderWithConnection };
```
Comment (radavis) [inline web/src/shared/apd-testing-library.js:59]: 🙌

File: web/src/containers/viewOnly/activities/Activity.js (hunk 1 above)
Comment (thetif) [reply web/src/containers/viewOnly/activities/Activity.js:245]: yep, it was a pretty easy fix, until I decided to add some other stuff in there with the testing. :D

Comment (radavis) [reply web/src/containers/viewOnly/activities/Activity.js:245]: I was curious how long this bug has been around: 9 months!

Comment (thetif) [reply web/src/containers/viewOnly/activities/Activity.js:245]: Yeah, we now have a test that will make sure that the tables are showing at least some of what they should

**Pull request merged**